powerscale/provider/nfs_export_settings_resource.go

```diff
@@ -664,6 +665,9 @@ Note that, NFS Export Settings is the native functionality of PowerScale. When c
 				Optional:            true,
 				Description:         "Specifies the maximum file size for any file accessed from the export. This parameter does not affect server behavior, but is included to accommodate legacy client requirements.",
 				MarkdownDescription: "Specifies the maximum file size for any file accessed from the export. This parameter does not affect server behavior, but is included to accommodate legacy client requirements.",
+				Validators: []validator.Int64{
+					int64validator.AtMost(9223372036854775807),
```

Comment:
the best practice when do release, we cannot use such big num const here

Reply:
ok. Thank you